test/k8sT/Services.go

```diff
@@ -567,7 +567,7 @@ var _ = SkipDescribeIf(helpers.RunsOn54Kernel, "K8sServicesTest", func() {
 		})
 	})
 
-	Context("Checks service across nodes", func() {
+	SkipContextIf(helpers.SkipQuarantined, "Checks service across nodes", func() {
```

Comment:
Since it's only the requests from outside the cluster that fail, could we skip those requests or, alternatively, only quarantine net-next since that's the only build for which we have those requests?